src/tools/ddsconf/CMakeLists.txt

```diff
@@ -37,4 +31,14 @@ else()
     "${CMAKE_CURRENT_BINARY_DIR}/include"
     $<BUILD_INTERFACE:$<TARGET_PROPERTY:ddsc,INCLUDE_DIRECTORIES>>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+
+  add_executable(${PROJECT_NAME}::ddsconf ALIAS ddsconf)
+
+  if (INSTALL_DDSCONF)
```

Comment:
Let's just always install this. It's not normally required by users, but it also can't do any harm.